### PR TITLE
Feature/deleting mocks

### PR DESCRIPTION
**Pull request by JJGothard**
Per [this JIRA ticket](https://postmanlabs.atlassian.net/browse/DEVREL-222) on the DevRel board, I updated "Setting up a mock server" to include deletion of a mock server. 

**Note:** The package-json file that was pushed to this branch was not changed by me; the changes that had been made recently we pulled into my branch when the origin was fetched. In triaging issues with running the learning center locally on my machine,   @ch264 and I went through this exercise (on another branch). It seemed to be the blocker on repeated issues with being able to lead the LC specifically, even when the rest of the site would run locally. Hope this makes sense :) 

Thanks!

### CODE REVIEW

**Comment by SueSmith** [comment]
@JJGothard Hey so we need to make sure your pull requests only include content changes and no changes to other files. Can you try updating your local branch from develop then running npm install, then push to github again? If changes to the file are still in the pr it'll need removed manually, I think you can do that in github desktop if I remember rightly.

**Comment by SueSmith** [comment]
(Removed from the pull request I mean, not removed from the repo.)

**Comment by JJGothard** [comment]
Ah, yes, yes, got it, @SueSmith 👍Should I cherry-pick this branch to have just the one file? Thanks!

**Comment by JJGothard** [comment]
@SueSmith , the extra file have been removed 🎉Ready when you are :)

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 1]
```diff
@@ -181,9 +181,9 @@ Your Postman account gives you a limited number of free mock server calls per mo
 
 You can check your usage limits in the [Postman API](https://docs.api.getpostman.com) or [account usage page](https://go.pstmn.io/postman-account-limits).
 
-## Editing mock servers
+## Editing and deleting mock servers
 
-You can edit existing mock servers in Postman since version `7.7.0`. The following properties of a mock server can be updated through the Postman app and web dashboard:
+You can edit or delete existing mock servers in Postman since version `7.7.0`. The following properties of a mock server can be updated through the Postman app and web dashboard:
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:186]: let's just lose the "since version" bit

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 2]
```diff
@@ -192,19 +192,27 @@ You can edit existing mock servers in Postman since version `7.7.0`. The followi
 
 ![Postman Mock Server edit options](https://assets.postman.com/postman-docs/mocks/mock-edit-modal.png)
 
-There are three ways to edit a mock server. You can edit from the Collection pane, through the **Mock Server** listing in **Browse** mode, or through the web dashboard of your workspace **Mock Servers** section.
+There are three ways to edit or delete a mock server: From the Collection pane, through the **Mock Server** listing in **Browse** mode, or through the web dashboard of your workspace **Mock Servers** section.
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:195]: Can we just reword this to "You can edit or delete a mock server from the..." and lose the "three ways" bit?

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 3]
```diff
@@ -192,19 +192,27 @@ You can edit existing mock servers in Postman since version `7.7.0`. The followi
...
 
-Editing mocks in Build mode:
+### Editing or deleting mocks in Build mode
 
-![Postman Mock Server edit in Build mode](https://assets.postman.com/postman-docs/mocks/mock-edit-build.png)
+* **To edit**: Click the pencil icon.
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:199]: let's keep bold primarily for ui items and make everything before a colon a complete sentence as per style guide - in this case I'd probably just lose the colon anyway

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 4]
```diff
@@ -192,19 +192,27 @@ You can edit existing mock servers in Postman since version `7.7.0`. The followi
 
 ![Postman Mock Server edit options](https://assets.postman.com/postman-docs/mocks/mock-edit-modal.png)
 
-There are three ways to edit a mock server. You can edit from the Collection pane, through the **Mock Server** listing in **Browse** mode, or through the web dashboard of your workspace **Mock Servers** section.
+There are three ways to edit or delete a mock server: From the Collection pane, through the **Mock Server** listing in **Browse** mode, or through the web dashboard of your workspace **Mock Servers** section.
 
-Editing mocks in Build mode:
+### Editing or deleting mocks in Build mode
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:197]: not sure we need headings for these when the sections are so short

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 5]
```diff
@@ -192,19 +192,27 @@ You can edit existing mock servers in Postman since version `7.7.0`. The followi
...
+* **To delete**: Click the trash icon.
 
-Editing mocks in Browse mode:
+![Postman Mock Server edit in Build mode](https://assets.postman.com/postman-docs/Delete+mock+build.jpg)
 
-![Postman Mock Server edit in Browse mode](https://assets.postman.com/postman-docs/mocks/mock-edit-browse.png)
+### Editing or deleting mocks in Browse mode
 
-Editing mocks from the Web Dashboard:
+To make your changes, click eithr **Edit** or **Delete**.
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:206]: typo "eithr"

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 6]
```diff
@@ -192,19 +192,27 @@ You can edit existing mock servers in Postman since version `7.7.0`. The followi
...
 
-![Postman Mock Server edit in Build mode](https://assets.postman.com/postman-docs/mocks/mock-edit-web-dashboard.png)
+![Postman Mock Server edit in Browse mode](https://assets.postman.com/postman-docs/Delte+mock+browse.jpg)
+
+### Editing or deleting mocks from the Web Dashboard
+
+1. At the top, click the **Mock servers** tab.
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:212]: whenever you're providing instructions for the web dashboard if you can include a link to it that would be great

Comment (SueSmith) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:212]: it looks like there's more than one way to navigate to mocks from the dashboard, screengrab looks like you've gone into a collection then navigated to mocks from there but the instructions are workspace > mock servers..? let's include both but make sure the instructions and screengrabs match

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 2 above)
Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:195]: Agree with Sue. Is this the same as the "build mode" referenced below? Let's try to use the same language. You could remove the subheaders below and just split this sentence into bullet points showing the multiple flows. 

I'd also split this section into ### Editing mock servers and ### Deleting mock servers

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 7]
```diff
@@ -192,19 +192,27 @@ You can edit existing mock servers in Postman since version `7.7.0`. The followi
...
+![Postman Mock Server edit in Build mode](https://assets.postman.com/postman-docs/Delete+mock+build.jpg)
 
-![Postman Mock Server edit in Browse mode](https://assets.postman.com/postman-docs/mocks/mock-edit-browse.png)
+### Editing or deleting mocks in Browse mode
 
-Editing mocks from the Web Dashboard:
+To make your changes, click eithr **Edit** or **Delete**.
 
-![Postman Mock Server edit in Build mode](https://assets.postman.com/postman-docs/mocks/mock-edit-web-dashboard.png)
+![Postman Mock Server edit in Browse mode](https://assets.postman.com/postman-docs/Delte+mock+browse.jpg)
```
Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:208]: I'd show the full app in this view so they can see "Browse" selected on the bottom and the header bar

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 8]
```diff
@@ -192,19 +192,27 @@ You can edit existing mock servers in Postman since version `7.7.0`. The followi
...
 
-Editing mocks in Build mode:
+### Editing or deleting mocks in Build mode
 
-![Postman Mock Server edit in Build mode](https://assets.postman.com/postman-docs/mocks/mock-edit-build.png)
+* **To edit**: Click the pencil icon.
+* **To delete**: Click the trash icon.
 
-Editing mocks in Browse mode:
+![Postman Mock Server edit in Build mode](https://assets.postman.com/postman-docs/Delete+mock+build.jpg)
```
Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:202]: I'd expand this a bit so users can see what's above "Documentation..Monitors..Mocks.." as well as the "History..Collections..APIs" labels

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 9]
```diff
@@ -192,19 +192,27 @@ You can edit existing mock servers in Postman since version `7.7.0`. The followi
...
-![Postman Mock Server edit in Browse mode](https://assets.postman.com/postman-docs/mocks/mock-edit-browse.png)
+### Editing or deleting mocks in Browse mode
 
-Editing mocks from the Web Dashboard:
+To make your changes, click eithr **Edit** or **Delete**.
 
-![Postman Mock Server edit in Build mode](https://assets.postman.com/postman-docs/mocks/mock-edit-web-dashboard.png)
+![Postman Mock Server edit in Browse mode](https://assets.postman.com/postman-docs/Delte+mock+browse.jpg)
+
+### Editing or deleting mocks from the Web Dashboard
```
Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:210]: -> Postman Dashboard or just Dashboard

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 10]
```diff
@@ -192,19 +192,27 @@ You can edit existing mock servers in Postman since version `7.7.0`. The followi
...
+
+1. At the top, click the **Mock servers** tab.
+2. Click either the **pencil** icon to edit or the **trash** icon to delete.
```
Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:213]: ~~either~~

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 1 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:186]: Done

[the remark this answers is not in the document]
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 2 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:195]: Done

Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 4 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Given Claire's above feedback about splitting them into sections above, I've kept the headers and introduced steps because in fact editing/deleting is a bit more involved than previously documented

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 3 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: This is now a step.

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 8 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: @claireriley , I've expanded the screenshot per this comment, but in general, I think we should all have a discussion around this topic as this seems to conflict with previous feedback I've been given about wanting to capture only the part of the screens that's relevant to the instructions the screenshot is illustrating. Also, in the interest of being consistent in the sections, I've applied your above feedback about splitting up the Editing and Deleting instructions into their own sections to this sections.

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 5 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 7 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: This section looks much different now since I wanted to make sure that if I split one of the subsections in two (one for Editing and one for Deleting) per the feedback above. I do like the idea of having the full app image to show Browse mode in the bottom, so I put that image in the first step for both editing and deleting. Let me know what you think.

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 9 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:235]: Done. Splitting this section as well for consistency.

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 10 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: This was removed in the process of splitting the section into "Editing" and "Deleting" :)

**Comment by JJGothard** [comment]
@SueSmith , all comments have been addressed. Also, when giving the content another pass, I made some changes that were consistent with conversations we've all had since the last time we were working in this PR. Thanks! 👍

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 6 above)
Comment (SueSmith) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:253]: Hm I think I'd try to reduce the repetition here along these lines:

* editing mocks from the dashboard
* deleting mocks from the dashboard

In each one I'd just start by saying navigate to a workspace by xyz, or a collection by xyz, then select your mock.. then just include the edit / delete steps once. Just now it reads like the same info is almost repeated four times..

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 11]
```diff
@@ -192,19 +192,77 @@ You can edit existing mock servers in Postman since version `7.7.0`. The followi
...
+![Postman Mock Server edit options](https://assets.postman.com/postman-docs/Edit+mocks+in+browse.jpg)
+
+### Deleting mocks in Browse mode
+
+1. At the top, click the **Mocks** tab.
+![Postman Mock Server edit options](https://assets.postman.com/postman-docs/Full+app+view+browse+mode.jpg)
+2. Next to the name of the mocks server, click the ellipsis **(...)** > **Delete**.
+![Postman Mock Server edit options](https://assets.postman.com/postman-docs/Deleting+mocks+from+Browse.jpg)
+3. Click **Delete**.
+![Postman Mock Server edit options](https://assets.postman.com/postman-docs/Delete+mock+modal.jpg)
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:229]: could we recrop this image? it's a little lop-sided atm, even if you just crop it inside the modal it'll look a wee bit cleaner

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 12]
```diff
@@ -192,19 +192,77 @@ You can edit existing mock servers in Postman since version `7.7.0`. The followi
 
 ![Postman Mock Server edit options](https://assets.postman.com/postman-docs/mocks/mock-edit-modal.png)
 
-There are three ways to edit a mock server. You can edit from the Collection pane, through the **Mock Server** listing in **Browse** mode, or through the web dashboard of your workspace **Mock Servers** section.
+In the Postman app, you can edit or delete a mock server from the **Collections** pane or from the **Mocks** listed in **Browse** mode. You can also edit or delete a mock server from the [Postman Dashboard](https://the.postman.co/workspaces?type=personal).
 
-Editing mocks in Build mode:
+### Editing mocks from the **Collections** pane
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:197]: if we can add the new subheadings to the toc plz

Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:205]: Done :)

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 11 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:237]: @SueSmith , Sure, do you mean the last image here? The one for step 3?

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 6 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:261]: Dashboard link: done (I'm assuming you don't mean in headers as well, yes?)

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 11 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:237]: I assumed step 3 cos "modal." Done :)

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 6 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Good call on the repeating sections: I implemented your strategy. I think I was trying to be consistent with previous feedback suggesting breaking things up. This does look better :) 

I also addressed to two avenues that lead to the mocks, but tried not to suggest using 'View all collections," per Sankalp's previous comment that this was not a recommended flow.

**Comment by JJGothard** [comment]
@SueSmith , all comments have been addressed :) Thanks!

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 13]
```diff
@@ -192,19 +198,61 @@ You can edit existing mock servers in Postman since version `7.7.0`. The followi
...
+![Postman Mock Server edit options](https://assets.postman.com/postman-docs/Deleting+mocks+from+Browse.jpg)
+3. Click **Delete**.
+![Postman Mock Server edit options](https://assets.postman.com/postman-docs/Confirm+mock+deletion+modal+.jpg)
+
+### Editing mocks from the Dashboard
+
+> To open the [Dashboard](https://www.postman.com/), sign in to Postman from your browser.
+
+1. Open the workspace or collection with the mock server you want to edit.
+    * If you have opened a workspace, click the **Mock servers** tab at the top of the page.
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:242]: let's simplify this further, instead of the "if you have opened..." list items just add a sentence after the "Open the workspace or..." sentence saying "Click **Mock servers** or **Mocks**..."

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 12 above)
Comment (SueSmith) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:203]: nbd but going forward let's not have bold in headings, it doesn't have an effect anyway

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 14]
```diff
@@ -181,9 +187,9 @@ Your Postman account gives you a limited number of free mock server calls per mo
 
 You can check your usage limits in the [Postman API](https://docs.api.getpostman.com) or [account usage page](https://go.pstmn.io/postman-account-limits).
 
-## Editing mock servers
+## Editing and deleting mock servers
 
-You can edit existing mock servers in Postman since version `7.7.0`. The following properties of a mock server can be updated through the Postman app and web dashboard:
+You can edit or delete existing mock servers in Postman. The following properties of a mock server can be updated through the Postman app and [Postman Dashboard](https://the.postman.co/workspaces?type=personal):
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:192]: just noticed another thing here - the dashboard link you've added is a link to your own personal dash - if you use the generic https://go.postman.co/ that should work

Comment (SueSmith) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:192]: or app.getpostman.com

Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:192]: Done. :)

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 12 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:203]: I agree -- that sounds good. I was also wondering about capitalizing "Collections" there... It's capitalized in the UI, but it makes for inconsistency in the content... I thought for later discussions, perhaps 🤪

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 13 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Done :)

**Comment by JJGothard** [comment]
@SueSmith , all comments have been addressed. Should be good to go 👌😊
